blog.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css">
+    <title>Adopt a Doggo</title>
+  </head>
+  <body>
+    <h1>ADOPTADOG</h1>
+    <section class="first">
+      <img src="images/adoptadog-logo.svg" height="250" width="250" alt="adoptadog logo">
+      <img src="images/header-image.jpg" height="400" width="1000" alt="header image">

```

Comment:
This isn't quite the best way to use the header image. We'll cover it later on in class, but if you want you can just delete this img tag. It's ok if it's just a blank header that just has a colored background